<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-12-18T11:00:47 IST" name="Suite" finished-at="2024-12-18T11:01:22 IST" duration-ms="35033">
    <groups>
      <group name="Purchase">
        <method signature="AppUsingPOM.OrderPlacement(java.util.HashMap)[pri:0, instance:rahulshettyacademy.SeleniumFramework.AppUsingPOM@49c90a9c]" name="OrderPlacement" class="rahulshettyacademy.SeleniumFramework.AppUsingPOM"/>
      </group> <!-- Purchase -->
    </groups>
    <test started-at="2024-12-18T11:00:47 IST" name="Test" finished-at="2024-12-18T11:01:22 IST" duration-ms="35033">
      <class name="rahulshettyacademy.SeleniumFramework.AppUsingPOM">
        <test-method is-config="true" signature="launchApp()[pri:0, instance:rahulshettyacademy.SeleniumFramework.AppUsingPOM@49c90a9c]" started-at="2024-12-18T11:00:48 IST" name="launchApp" finished-at="2024-12-18T11:00:53 IST" duration-ms="5081" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method signature="OrderPlacement(java.util.HashMap)[pri:0, instance:rahulshettyacademy.SeleniumFramework.AppUsingPOM@49c90a9c]" started-at="2024-12-18T11:00:53 IST" name="OrderPlacement" data-provider="getData" finished-at="2024-12-18T11:01:17 IST" duration-ms="23975" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{productname=ZARA COAT 3, password=Shivi@0881, username=shivangiag004@gmail.com}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderPlacement -->
        <test-method is-config="true" signature="driverClose()[pri:0, instance:rahulshettyacademy.SeleniumFramework.AppUsingPOM@49c90a9c]" started-at="2024-12-18T11:01:17 IST" name="driverClose" finished-at="2024-12-18T11:01:17 IST" duration-ms="308" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- driverClose -->
        <test-method is-config="true" signature="launchApp()[pri:0, instance:rahulshettyacademy.SeleniumFramework.AppUsingPOM@49c90a9c]" started-at="2024-12-18T11:01:17 IST" name="launchApp" finished-at="2024-12-18T11:01:21 IST" duration-ms="3932" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApp -->
        <test-method signature="OrderPlacement(java.util.HashMap)[pri:0, instance:rahulshettyacademy.SeleniumFramework.AppUsingPOM@49c90a9c]" started-at="2024-12-18T11:01:21 IST" name="OrderPlacement" data-provider="getData" finished-at="2024-12-18T11:01:22 IST" duration-ms="808" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{productname=ADIDAS ORIGINAL1, password=Shivi@0881, username=shivangiag004@gmail.com}]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Product with name 'ADIDAS ORIGINAL1' is not present.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Product with name 'ADIDAS ORIGINAL1' is not present.
at rahulshettyacademy.AppLocators.ProductPageHandler.getProduct(ProductPageHandler.java:48)
at rahulshettyacademy.SeleniumFramework.AppUsingPOM.OrderPlacement(AppUsingPOM.java:32)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- OrderPlacement -->
        <test-method is-config="true" signature="driverClose()[pri:0, instance:rahulshettyacademy.SeleniumFramework.AppUsingPOM@49c90a9c]" started-at="2024-12-18T11:01:22 IST" name="driverClose" finished-at="2024-12-18T11:01:22 IST" duration-ms="222" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- driverClose -->
      </class> <!-- rahulshettyacademy.SeleniumFramework.AppUsingPOM -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
